README.md

```diff
@@ -1,4 +1,4 @@
-# Mor<a name="readme-top">School-library</a>
+<a name="readme-top">School-library</a>
```

Comment:
Hi @alicemirigo92 
You have done a great job implementing some of the requested changes mentioned by the previous code reviewer but you still have some issues to fix in your README file as asked.

- [ ] Kindly update the `Getting Started` section of your README file with the different steps to run your project locally.